state/executor_test.go

```diff
@@ -39,7 +39,7 @@ func TestCreateBlock(t *testing.T) {
 	nsID := [8]byte{1, 2, 3, 4, 5, 6, 7, 8}
 
 	mpool := mempoolv1.NewTxMempool(logger, cfg.DefaultMempoolConfig(), proxy.NewAppConnMempool(client), 0)
-	executor := NewBlockExecutor([]byte("test address"), nsID, "test", mpool, proxy.NewAppConnConsensus(client), nil, logger)
+	executor := NewBlockExecutor([]byte("test address"), nsID, "test", mpool, proxy.NewAppConnConsensus(client), false, nil, logger)
```

Comment:
agree. this will require slightly more tests to accommodate both cases as the assertions following the executor initialization will drastically change when the fraud proof is enabled vs disabled. will add the detailed tests.

Reply:
actually it didn't require redoing the tests. simple change to expand it. ready for re-review @tzdybal @Manav-Aggarwal